src/main/java/io/stargate/sgv2/jsonapi/config/OperationsConfig.java

```diff
@@ -252,6 +252,6 @@ interface ConsistencyConfig {
     }
   }
   /** @return Flag to enable server side vectorization. */
-  @WithDefault("false")
+  @WithDefault("true")
```

Comment:
This by default has to be false. If you need it true for any testing add test profile like `EnabledVectorizeProfile`

Reply:
I set it to true for testing purposes. Forgot to remove it